File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Prices/IPriceData.php
<?php

namespace Ohtap\ECommerce\DomainModels\Objects\Prices;


use  Ohtap\ECommerce\DomainModels\Core\Object as  CoreObjects ;


interface IPriceData extends IPriceParent,CoreObjects\IObjectID, CoreObjects\IObjectCode,
    CoreObjects\IObjectDescription, CoreObjects\IObjectName, CoreObjects\IObjectOrder, CoreObjects\IObjectState, CoreObjects\IObjectTitle
{

}

File: Ohtap E Commerce Domain Models/Development/NetCore/Models Objects/Models Objects/Prices/IPriceData.cs
using Ohtap.ECommerce.Domain.Models.Objects.Datasets;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ohtap.ECommerce.Domain.Models.Objects.Currencies;
using Ohtap.ECommerce.Domain.Models.Objects.Regions;
using CoreObject = Ohtap.ECommerce.Domain.Models.Core.Object;

namespace Ohtap.ECommerce.Domain.Models.Objects.Prices
{
    public interface IPriceData :  CoreObject.IObjectID, CoreObject.IObjectCode, CoreObject.IObjectName, CoreObject.IObjectOrder, CoreObject.IObjectDescription, CoreObject.IObjectState, IPriceValue

    {
        IDataItemList<IDataItemData> Dataset { get; set; }
        public CurrencyBasic Currency { get; set; }
    }
}
